pkg/ai/bedrock.go

```diff
@@ -0,0 +1,35 @@
+package ai
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/bedrockagentruntime"
+	"github.com/aws/aws-sdk-go-v2/service/bedrockagentruntime/types"
+)
+
+// InvokeAWSBedrockAgent invokes the AWS Bedrock agent with the provided parameters.
+func InvokeAWSBedrockAgent(cfg aws.Config, ctx context.Context, agentID, inputText, sessionID string) (string, error) {
+	bedrockAgentRuntime := bedrockagentruntime.NewFromConfig(cfg)
```

Comment:
I think you want to wrap this in a struct so we can batch requests against 1 agent instead of recreating the representation each time.